bedrock/externalpages/templates/externalpages/analytics/index.html

```diff
@@ -0,0 +1,20 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{# This file is used only for automated testing for google tag manager #}
```

Comment:
Suggestion (non-blocking): my initial though was this could be useful information to display in the web page itself, as opposed to a comment only visible for devs? Up to you here though, either way is enough I would think 👍